### PR TITLE
Fix for the register external store

### LINKED ISSUE #74: External store register fails with content-type exception
External store register fails with content-type exception

### PR DESCRIPTION
**Pull request by svt-hchaudhari**
Signed-off-by: Hemant Chaudhari <hchaudhari@hpe.com>

### Description
Fix for the register external store

### Issues Resolved
Fixes #74 

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: simplivity/resources/external_stores.py [hunk 1]
```diff
@@ -106,7 +106,8 @@ def register_external_store(self, management_ip, name, cluster, username, passwo
             cluster = clusters_obj.get_by_name(cluster)
 
         data['omnistack_cluster_id'] = cluster.data['id']
-        self._client.do_post(URL, data, timeout)
+        custom_headers = {'Content-type': 'application/vnd.simplivity.v1.14+json'}
```
Comment (svt-kmcdaniel) [inline simplivity/resources/external_stores.py]: Is this the minimum supported content type?

Comment (svt-hchaudhari) [reply simplivity/resources/external_stores.py:109]: Yes this the minimum supported content-type for API

Comment (svt-hchaudhari) [reply simplivity/resources/external_stores.py]: Minimum version for the API is the ''application/vnd.simplivity.v1.11+json'', but in swagger the default version selected in curl command is ''application/vnd.simplivity.v1.14+json'' therefore we used this version.

Comment (svt-hchaudhari) [reply simplivity/resources/external_stores.py]: Verified and tested with the minimum version ''application/vnd.simplivity.v1.11+json'' API works fine. Therefore updated the content-type to minimum version
Thanks @svt-kmcdaniel

**Review by sijeesh — APPROVED** [review]
LGTM 👍

**Pull request merged**